contract/src/main/java/com/ritense/valtimo/contract/web/rest/error/ExceptionTranslator.java

```diff
@@ -65,9 +65,11 @@ public ResponseEntity<Problem> process(@Nullable ResponseEntity<Problem> entity,
         }
 
         final String msg = "message";
-        if (problem instanceof ConstraintViolationProblem) {
+        if (problem instanceof ConstraintViolationProblem constraintViolationProblem) {
             builder
-                .with("violations", ((ConstraintViolationProblem) problem).getViolations())
+                .with("errors", constraintViolationProblem.getViolations().stream()
```

Comment:
Not sure i'm happy with this. This is a breaking change that will impact any @valid we use in the codebase. Do we want to do this?

Reply:
This code exclusively influences the HTTP response generated by @Valid. Currently, the application fails to correctly display this error. In instances where there are multiple violations in a single response, the frontend anticipates them to be in the errors field.

We could still keep the `violations` field to make it non-breaking. But I haven't seen anyone use it.